sdk/tests/deqp/functional/gles3/es3fFboRenderTest.js

```diff
@@ -1007,6 +1009,7 @@ goog.scope(function() {
         rrUtil.drawQuad(
             context, texShaderID, [-1.0, -1.0, 0.0], [1.0, 1.0, 0.0]
         );
+        context.disable(gl.BLEND);
```

Comment:
Blending should not always be enabled, or it will let the following tests go wrong